src/index.html

```diff
@@ -1,14 +1,55 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#main" class="logo">
+        <img src="images/logo.png" alt="logo" >
+      </a>
+      <nav class="nav">
+        <ul class="nav-links">
+          <li class="nav-link">
+            <a href="#apple" class="link is-active">APPLE</a>
+          </li>
+          <li class="nav-link">
+            <a href="#samsung" class="link">SAMSUNG</a>
+          </li>
+          <li class="nav-link">
+            <a href="#smartphones" class="link">SMARTPHONES</a>
+          </li>
+          <li class="nav-link">
+            <a href="#laptops" class="link"
```

Comment:
Please re-read point 13 in checklist
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/38f67c0d-9803-4b24-9912-96b13a56d107)